SlackAPI.Tests/Configuration/IntegrationFixture.cs

```diff
@@ -55,8 +56,10 @@ private SlackSocketClient GetClient(string authToken)
 
             using (var syncClient = new InSync($"{nameof(SlackClient.Connect)} - Connected callback"))
             using (var syncClientSocket = new InSync($"{nameof(SlackClient.Connect)} - SocketConnected callback"))
+            using (var syncClientSocketHello = new InSync($"{nameof(SlackClient.Connect)} - SocketConnected hello callback"))
             {
                 client = new SlackSocketClient(authToken);
+                client.OnHello += () => syncClientSocketHello.Proceed();
```

Comment:
Adding this extra wait fixes some false positive failures when I was testing the implementation